packages/clerk-js/src/ui/components/SignIn/SignIn.tsx

```diff
@@ -44,9 +44,8 @@ function SignInRoutes(): JSX.Element {
           <SignInSSOCallback
             signUpUrl={signInContext.signUpUrl}
             signInUrl={signInContext.signInUrl}
-            afterSignInUrl={signInContext.afterSignInUrl}
-            afterSignUpUrl={signInContext.afterSignUpUrl}
-            redirectUrl={signInContext.redirectUrl}
+            signInForceRedirectUrl={signInContext.afterSignInUrl}
+            signUpForceRedirectUrl={signInContext.afterSignUpUrl}
             continueSignUpUrl={signInContext.signUpContinueUrl}
```

Comment:
I am unsure of the API we want her - this is the component that calls `Clerk.handleRedirectCallback` internally so even if this is an internal change, the API change is still reflected  on `Clerk.handleRedirectCallback` 

Does this function need all 4 URL options or can we simply pass `redirectUrl`?